pavarde.php

```diff
@@ -20,6 +20,8 @@
 </form>
     <?php
 } else {
+    $_GET['vardas'] = str_replace(" ", ",.", $_GET['vardas']);
```

Comment:
Jau čia blogai. CSV faile kableliais atskiriami atskiri informacijos vienetai (pvz vardas; pvz pavardė, pvz amžius). Pačių vienetų - niekaip keisti paprastai nereikia (buvo informacijos vienete tarpų - ir palik)
O skaitant CSV failą irgi laikaisi to paties principo - suskaidai pagal kablelius - gauni informacijos vienetus (gal su tarpais viduje, o gal ir ne - tau nesvarbu)